src/Ctype/bootstrap.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Ctype as p;
+
+if (!extension_loaded('ctype')) {
+    function ctype_alnum($text) { return p\Ctype::ctype_alnum($text); }
```

Comment:
Do we want individual function_exists/ is_callable checks for each function?
In case they have their own polyfills somewhere?

Reply:
good question :)
in other polyfills, we don't use extension_loaded but function exists, but we do only one check
so the above if should be eg `if (!function_exists('ctype_alnum')) {`
we'll consider that if ppl have one, they have them all.
If not, issue is on their side

Reply:
:+1: I'll go ahead and change that